metabrainz/musicbrainz-server
# MBS-13561 / MBS-13571 / MBS-13818 / MBS-13580 / MBS-13825: Show primary aliases in more places

### Implement MBS-13561 / MBS-13571 / MBS-13818 / MBS-13580 / MBS-13825

# Description
Extends the code @derat added in MBS-7646 to show primary aliases in a lot more places (when they differ from the entity name). See commits and tickets for details.

There's two cases where this intentionally skips showing primary aliases: instruments and country areas (which we translate separately, so in most cases this would just cause unwanted redundancy otherwise).

# Testing
Manually, navigating all around the site and trying to see what places need to separately load the aliases.

Places where this is known to do nothing yet: TT-based places, mostly, plus I didn't implement any way to show recording primary aliases in the release relationship editor.

Places where I tested things should work fine (but it's not impossible I missed some edge cases for some entities, which I expect beta testers would let me know about): 
* Every aliasable entity's header
* Tag pages
* Rating pages
* Entities in a series
* Entities in a collection
* Entities related to the one being shown
* Artist credits everywhere I checked (showing on hover, as with sort name and disambiguation)
* Search results (both direct and indexed)
* Writers in work lists (including search)
* Per-entity lists in artist pages, area pages, instrument pages, event pages
* Merge queue on footer
* Merge pages
* Edit displays
* Genre list
* Under each track in the release page (when logged in, if the recording primary alias does not match the track name)

I'm going to put this up on test and let people have a go at finding places I forgot (and let @Aerozol critique it), but it seems trivial to add more places later, so I don't think we need to be super careful with it right now (more important is to test nothing broke while adding the aliases).
